src/components/FiltersPanel/FiltersPanel.tsx

```diff
@@ -207,21 +120,20 @@ export const FiltersPanel: FC<{
                 {children}
                 <FiltersBarItem layout="fill">
                     <FiltersBarControlGroup>
-                        {isFiltersEmpty ? (
-                            <Button
-                                key="filter"
-                                ref={filterTriggerRef}
-                                text={tr('Filter')}
-                                onClick={() => setFilterVisible((val) => !val)}
-                                iconLeft={<IconAddOutline size="xxs" />}
-                            />
-                        ) : (
+                        {nullable(
+                            isFiltersEmpty && !groupedByProject,
```

Comment:
new business logic? Are we really going to disable ability to apply filters for grouped view?